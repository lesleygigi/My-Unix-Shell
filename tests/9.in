ls tests/test_dir
