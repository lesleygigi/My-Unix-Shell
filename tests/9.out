ls tests/test_dir
file1
file2
file3
file4
